src/store/index.ts

```diff
@@ -264,6 +266,16 @@ const {
     setDirectionalEdges(state, directionalEdges: boolean) {
       state.directionalEdges = directionalEdges;
     },
+
+    updateSimulationForce(state: State, payload: {
+      forceType: 'center' | 'charge' | 'link' | 'collision';
+      forceValue: ForceCenter<Node> | ForceManyBody<Node> | ForceLink<Node, SimulationLink> | ForceCollide<Node>;
+    }) {
```

Comment:
Why a `payload` object, rather than `forceType` and `forceValue` (and why not simply `type` and `value`)?

Reply:
It's a payload because the vuex mutations can only take one parameter (see [here](https://vuex.vuejs.org/guide/mutations.html#commit-with-payload)).

I called it `forceType` and `forceValue`, just to be explicit about what we're receiving. If you'd like me to rename them, that's fine. Should be a super quick fix.

Reply:
> It's a payload because the vuex mutations can only take one parameter (see [here](https://vuex.vuejs.org/guide/mutations.html#commit-with-payload)).

Ah right, thanks for reminding me.

> I called it `forceType` and `forceValue`, just to be explicit about what we're receiving. If you'd like me to rename them, that's fine. Should be a super quick fix.

Totally up to you. Since the commit function has "force" in it, the word "force" seems redundant in the payload field names, but it really doesn't matter a lot.

Reply:
Let's leave it as is for now